tools/xchain-suite/src/components/swap/SwapTrackingModal.tsx

```
@@ -251,6 +251,8 @@ export function SwapTrackingModal({
     ? `https://scan.chainflip.io/channels/${depositChannelId || ''}`
     : protocol === 'Thorchain'
     ? `https://track.thorchain.org/${txHash}`
+    : protocol === 'OneClick'
```

ESLint

[error] Insert `··`

(prettier/prettier)